tox.ini

```diff
@@ -29,7 +29,7 @@ commands =
 
 [testenv:bandit]
 deps = bandit
-commands = bandit -r --ini tox.ini
+commands = bandit -r --ini tox.ini behave_django
```

Comment:
Not sure why we're checking the whole project, specially since most of the issue it finds are related to `site-packages/os.py` and the likes.

Reply:
After all the tests passed, i tried moving this back to how it was and it keeps throwing exit code 1 for code thats not in this repo... im very confused.

Reply:
The target is [covered by](https://github.com/behave/behave-django/blob/master/tox.ini#L67) the `[bandit]` section.

Not sure where you have your site-packages (Is this visible as an issue in the CI jobs?). They shouldn't be in your project folder. If you have a `.venv` in your project folder (which is a bad habit I see often) move it somewhere else, that should fix it for your local development

Reply:
The (performance) issue with Bandit is a [known issue](https://github.com/PyCQA/bandit/issues/490), unfortunately unresolved. The solution is to [use an older version of Bandit](https://github.com/bittner/pyclean/blob/master/tox.ini#L23) that does not have the issue, e.g.
```ini
deps = bandit<1.6.0
```